build/package/Symbols.targets

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <SymbolsProjectFileName>$(MSBuildThisFileDirectory)/Symbols.csproj</SymbolsProjectFileName>
+    <SymbolsNupkgPackageId>sdk.$(ProductMonikerRid).Microsoft.DotNet.SDK</SymbolsNupkgPackageId>
```

Comment:
Interesting... in the `runtime.{rid}.Microsoft...` packages I've always thought of `runtime` just indicating the package was runtime-specific, not that it's part of the Runtime. (So I was expecting a `runtime.` prefix here, too.) I'm not sure what's correct. In the end though, this name is arbitrary as long as it's only used for symbol packages.

Reply:
@dagood 
I can change this to "runtime" - I'm not sure what's correct here either...

Reply:
Thanks. `runtime` makes more sense to me, but if this ends up confusing someone I'm fine with changing it back to `sdk`. Something I remembered that makes me lean more this way is that the NuGet RID graph feature uses `runtime.json` files, which seems to favor `runtime` being a reference to the NuGet functionality rather than the product the package is a part of.